pkg/testsuite/clients.go

```diff
@@ -36,38 +36,39 @@ type AdminClient struct {
 }
 
 // BatchIssueCode calls the issue-batch API call.
-func (c *AdminClient) BatchIssueCode(req api.BatchIssueCodeRequest) (*api.BatchIssueCodeResponse, error) {
+func (c *AdminClient) BatchIssueCode(req api.BatchIssueCodeRequest) (int, *api.BatchIssueCodeResponse, error) {
```

Comment:
Thinking out loud... should we return a different type here?

```go
type ImBadAtNaming struct {
  Successes int
  Failures int
  Responses []*api.IssueCodeResponse
```

Reply:
we could, but I want to return relatively raw data here because I want to validate and traverse it as an external caller might